frontend/packages/console-app/console-extensions.json

```diff
@@ -319,6 +320,15 @@
     }
   },
 
+  {
```

Comment:
is this piece necessary?

Reply:
it is, we needed to remove the oauth details page so it only part of console the the oauth api is installed on the cluster.